omod/src/main/webapp/resources/scripts/attachments/directives/fileUpload.js

```diff
@@ -106,12 +106,7 @@ angular
                                             'thumbnailHeight' : 100,
                                             'thumbnailWidth' : 100,
                                             'maxFiles' : 1,
-                                            'autoProcessQueue' : false,
-                                            'renameFilename' : function(name) {
```

Comment:
What about the other things this is doing other than adding timestamp @mogoodrich ?

Reply:
Oh, fair good catch.

Reply:
It's removing the extension (and then it gets added back in at the end)